tests/testthat/test_dim_reduction.R

```diff
@@ -86,4 +163,18 @@ test_that('PCA produces the correct output',{
   dimmer_g <- dim_reduction(ldata_g, k = 2)
   expect_identical(dimmer_g, standard_g)
   
+  ## seqdata ##
+  set.seed(338)
+  expect_warning(
+    dimmer <- dim_reduction(seqdata, k = 3),
+    paste("group_designation has not been run on this data and may limit",
+          "plotting options",
+          sep = " ")
+  )
+  expect_identical(dimmer, seq_standard)
```

Comment:
I am wondering what is actually being tested in these, since the test code basically just re-runs the code inside the function.

I guess it checks that it correctly uses glmpca for seqData, which is probably fine.  Just trying to think if there's something else that can be tested here, possibly not since these really are just wrappers for pcaMethods and glmpca

Reply:
@clabornd you are correct that it is testing mostly the same stuff, idea being its structured the same way as lines 44-63. We could likely pull these out as a static unit test df and compare that way for both if preferable?

Reply:
Sry not sure I follow your last sentence, but sounds like we're merging soon and this wasn't a red flag or anything.